### PR TITLE
Teams: Add People dialog

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

Here's the PR for adding users to a team with a specified role.

Changes:

* This is mostly just doing a searchv3 with a role dropdown next to it, and adding the selected users to a team.  But!
* We want to show users already in the team with a checkmark and make them unselectable. So we need an efficient way to query team membership across a list of search results.  That way is the new `entities.teams.teamNameToMemberUsernames`, and the results list takes a new `disableIfInTeamName={teamname}` prop that tests result rows against `teamname`'s members.
* A team can't have more than 20 members right now.  So if the sum of the existing members and the ones you've got chosen as pills in the search query would take you over 20, you get a banner.
* On mobile, our `<Dropdown />` doesn't work so you can't change the role away from the default (`writer`) to something else right now.  Open to suggestions on what to use instead.  I think new Git repos are broken on mobile for the same reason?  Maybe we should just fix `<Dropdown />` soon?

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 65de1f08f69b...090898031237 on linux. <details><summary>:mag_right: 22 errored, 2 new, 6 removed, 27 changed</summary>

 * Error in 090898031237: **You_Rekey-Normal-ERROR**  
   You_Rekey-Normal-ERROR rendered
 * Error in 65de1f08f69b: **Devices_Devices_List-Devices-ERROR**  
   Devices_Devices_List-Devices-ERROR rendered
 * Error in 090898031237: **Devices_Devices_List-Devices-ERROR**  
   Devices_Devices_List-Devices-ERROR rendered
 * Error in 65de1f08f69b: **Chat_List-Empty-ERROR**  
   Chat_List-Empty-ERROR rendered
 * Error in 65de1f08f69b: **Chat_Input-Normal-ERROR**  
   Chat_Input-Normal-ERROR rendered
 * Error in 090898031237: **Chat_Input-Normal-ERROR**  
   Chat_Input-Normal-ERROR rendered
 * Error in 65de1f08f69b: **Chat_Inbox-You_Rekey_Selected-ERROR**  
   Chat_Inbox-You_Rekey_Selected-ERROR rendered
 * Error in 090898031237: **Chat_Inbox-You_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-You_Rekey_Selected-ERROR-090898031237.png)
 * Error in 65de1f08f69b: **Chat_Inbox-You_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-You_Rekey-ERROR-65de1f08f69b.png)
 * Error in 090898031237: **Chat_Inbox-You_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-You_Rekey-ERROR-090898031237.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Selected_Normal-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Selected_Normal-ERROR-65de1f08f69b.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Selected_Muted-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Selected_Muted-ERROR-65de1f08f69b.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Part_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Part_Rekey_Selected-ERROR-65de1f08f69b.png)
 * Error in 090898031237: **Chat_Inbox-Part_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Part_Rekey_Selected-ERROR-090898031237.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Part_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Part_Rekey-ERROR-65de1f08f69b.png)
 * Error in 090898031237: **Chat_Inbox-Part_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Part_Rekey-ERROR-090898031237.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Normal-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Normal-ERROR-65de1f08f69b.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Long_Top-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Long_Top-ERROR-65de1f08f69b.png)
 * Error in 090898031237: **Chat_Inbox-Long_Top-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Long_Top-ERROR-090898031237.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Long_Bottom-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Long_Bottom-ERROR-65de1f08f69b.png)
 * Error in 090898031237: **Chat_Inbox-Long_Bottom-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Long_Bottom-ERROR-090898031237.png)
 * Error in 65de1f08f69b: **Chat_Inbox-Empty-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Empty-ERROR-65de1f08f69b.png)
 * Added: **Chat_List-Empty** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_List-Empty-090898031237.png)
 * Added: **Chat_Inbox-Empty** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Inbox-Empty-090898031237.png)
 * Changed: **Colors-grey_cccccc** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-grey_cccccc-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-grey_cccccc-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-grey_cccccc.png)
 * Changed: **Colors-dark_Blue_4_103_c_64** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-dark_Blue_4_103_c_64-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-dark_Blue_4_103_c_64-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-dark_Blue_4_103_c_64.png)
 * Changed: **Colors-dark_Blue_3_0_a_3052** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-dark_Blue_3_0_a_3052-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-dark_Blue_3_0_a_3052-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-dark_Blue_3_0_a_3052.png)
 * Changed: **Colors-black_10_rgba_0_0_0_0_10** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-black_10_rgba_0_0_0_0_10-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-black_10_rgba_0_0_0_0_10-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Colors-black_10_rgba_0_0_0_0_10.png)
 * Changed: **Checkbox-Disabled_unchecked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Checkbox-Disabled_unchecked-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Checkbox-Disabled_unchecked-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Checkbox-Disabled_unchecked.png)
 * Changed: **Checkbox-Disabled_checked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Checkbox-Disabled_checked-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Checkbox-Disabled_checked-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Checkbox-Disabled_checked.png)
 * Changed: **Buttons-Unfollow** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Unfollow-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Unfollow-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Unfollow.png)
 * Changed: **Buttons-Secondary_small** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Secondary_small-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Secondary_small-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Secondary_small.png)
 * Changed: **Buttons-Primary_waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_waiting-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_waiting-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_waiting.png)
 * Changed: **Buttons-Primary_full_Width** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_full_Width-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_full_Width-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_full_Width.png)
 * Changed: **Buttons-Primary_disabled** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_disabled-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_disabled-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Primary_disabled.png)
 * Changed: **Buttons-Follow_small_waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Follow_small_waiting-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Follow_small_waiting-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Follow_small_waiting.png)
 * Changed: **Buttons-Follow_small** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Follow_small-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Follow_small-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Follow_small.png)
 * Changed: **Buttons-Danger_full_Width** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Danger_full_Width-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Danger_full_Width-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Danger_full_Width.png)
 * Changed: **Buttons-Danger** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Danger-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Danger-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Buttons-Danger.png)
 * Changed: **Avatar-Normal_x_80** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_80-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_80-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_80.png)
 * Changed: **Avatar-Normal_x_64** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_64-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_64-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_64.png)
 * Changed: **Avatar-Normal_x_24** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_24-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_24-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Normal_x_24.png)
 * Changed: **Avatar-Mutual_Follow_x_48** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Mutual_Follow_x_48-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Mutual_Follow_x_48-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Mutual_Follow_x_48.png)
 * Changed: **Avatar-Mutual_Follow_x_176** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Mutual_Follow_x_176-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Mutual_Follow_x_176-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Mutual_Follow_x_176.png)
 * Changed: **Avatar-Follows_You_x_80** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Follows_You_x_80-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Follows_You_x_80-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Follows_You_x_80.png)
 * Changed: **Avatar-Follows_You_x_16** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Follows_You_x_16-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Follows_You_x_16-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Follows_You_x_16.png)
 * Changed: **Avatar-Following_x_32** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_32-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_32-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_32.png)
 * Changed: **Avatar-Following_x_24** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_24-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_24-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_24.png)
 * Changed: **Avatar-Following_x_176** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_176-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_176-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Following_x_176.png)
 * Changed: **Avatar-Fallback_x_48** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Fallback_x_48-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Fallback_x_48-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Fallback_x_48.png)
 * Changed: **Avatar-Fallback_x_40** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Fallback_x_40-65de1f08f69b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Fallback_x_40-090898031237.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Avatar-Fallback_x_40.png)
 * Removed: **Chat_Banner_Invite-Invite** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Banner_Invite-Invite-65de1f08f69b.png)
 * Removed: **Chat_Banner_Info-Info** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Banner_Info-Info-65de1f08f69b.png)
 * Removed: **Chat_Banner_Error-Error** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Banner_Error-Error-65de1f08f69b.png)
 * Removed: **Chat_Banner_Broken-Broken_Tracker_3** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Banner_Broken-Broken_Tracker_3-65de1f08f69b.png)
 * Removed: **Chat_Banner_Broken-Broken_Tracker_2** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Banner_Broken-Broken_Tracker_2-65de1f08f69b.png)
 * Removed: **Chat_Banner_Broken-Broken_Tracker_1** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507702425049-65de1f08f69b-090898031237-linux/Chat_Banner_Broken-Broken_Tracker_1-65de1f08f69b.png)
</summary>

**Comment by cjb** [comment]
@keybase/react-hackers Ready for review now!  Removed the 20 person team limit, it was an outdated design.  And we're leaving the `<Dropdown />` in for this PR and will fix it in a future one.

File: shared/teams/add-people/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+// @flow
+import * as React from 'react'
+import {Box, Button, Dropdown, ProgressIndicator, Text, PopupDialog} from '../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
+import capitalize from 'lodash/capitalize'
+import {isMobile} from '../../constants/platform'
+import UserInput from '../../search/user-input/container'
+import SearchResultsList from '../../search/results-list/container'
+
+const MaybePopup = isMobile
```
Comment (chrisnojima) [inline shared/teams/add-people/index.js]: This is a good example of the lack of a solution for desktop and mobile pop up/modal split. 

We should think of a good abstraction that solves this for us. We have a ton of this kind of solution that’s similar but slightly different. It’s likely worth investing in this next sprint. 

Would love to hear people’s ideas. Maybe something route tree solves for us?

File: shared/teams/add-people/index.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        <Text type="Body">{capitalize(item)}</Text>
+      </Box>
+    )
+  }
+
+  _makeDropdownItems = () => ['admin', 'owner', 'reader', 'writer'].map(item => this._makeDropdownItem(item))
+
+  _dropdownChanged = (node: React.Node) => {
+    // $FlowIssue doesn't understand key will be string
+    const selectedRole: string = (node && node.key) || null
```
Comment (chrisnojima) [inline shared/teams/add-people/index.js]: The or null likely also makes flow unhappy

**Pull request merged**